test/cbor.spec.js

```diff
@@ -71,52 +70,41 @@ describe('IPLD -> CBOR', () => {
     )
   })
 
-  it('@link, with properties', () => {
+  it('/, with properties', () => {
     const src = {
       data: 'hello world',
       size: 11,
       secret: {
-        '@link': 'hello-world',
+        '/': 'hello-world',
         secret: 3
```

Comment:
to make it more memorable. That is the point of the test, isn't it? Links should not share props in their obj